src/module/apps/hotbar.js

```diff
@@ -26,7 +26,25 @@ class PTUHotBar extends Hotbar {
                     if (!item) return;
 
                     if (item.type === "effect" || item.type === "condition") {
-                        const command = `const item = await fromUuid("${data.uuid}");await item?.apply?.(game.user.targets.size > 0 ? [...game.user.targets] : (canvas.tokens.controlled ?? []), "${data.uuid}")`;
+                        const command = `const actors = canvas.tokens.controlled.flatMap((token) => token.actor ?? []);
+if (actors.length === 0 && game.user.character) actors.push(game.user.character);
+if (actors.length === 0) {
+    return ui.notifications.error("PTU.Notifications.ApplyEffectNoTargetSelected", { localize: true });
+}
+
+const ITEM_UUID = "${data.uuid}"; // ${item.name}
+const source = (await fromUuid(ITEM_UUID)).toObject();
```

Comment:
Why are we switching between `UPPER_CASE` and what I pressume to be `camelCase` between variables?
We use camelCase everywhere else so lets keep that consistent.